challenges/1-media-alunos/js/celsonf/mediaAlunos.js

```diff
@@ -0,0 +1,6 @@
+function mediaAlunos(primeiraNota,segundaNota){
+    let media;
+    return media = (primeiraNota + segundaNota) / 2;
+    }
```

Comment:
Remover Identação do fechamento de chaves
```suggestion
}
```